staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -626,6 +629,45 @@ func (r *Request) Stream() (io.ReadCloser, error) {
 	}
 }
 
+// assembles a User-Agent header that contains the br.client.transport().agent acktrace of function calls
+func appendDebugBacktrace(currUserAgent string) string {
+	var lines []string
+	var callers []uintptr
+	const SKIP_CALLERS = 3 // TODO: is this the right hard-coded value?
+
+	callers = make([]uintptr, 64)
+	num_callers := goruntime.Callers(SKIP_CALLERS, callers)
+
+	cwdir, err := os.Getwd()
+	if err != nil {
+		return fmt.Sprintf("%v",err)
+	}
+
+	if num_callers > 0 {
+		frames := goruntime.CallersFrames(callers)
```

Comment:
https://golang.org/pkg/runtime/#CallersFrames

Provides the frames for stacktrace